src/main/java/com/ericsson/ei/controller/model/Subscription.java

```diff
@@ -153,6 +159,26 @@ public void setUserName(String userName) {
         this.userName = userName;
     }
 
+    @JsonProperty("restUser")
```

Comment:
Use lombok lib for getter and setter, something like
@JsonProperty("token")
@ Getter @ Setter
private String token;

Reply:
token is already removed